src/main/charts/confluence/values.yaml

```diff
@@ -761,6 +766,10 @@ synchrony:
     #
     loadBalancerIP:
 
+    # -- Additional annotations to apply to Synchrony Service
```

Comment:
I don't think they are additional, it is `either or`. If defined, they are used, otherwise Confluence ones are used.